src/editors/containers/ProblemEditor/components/EditProblemView/SettingsWidget/messages.js

```diff
@@ -0,0 +1,154 @@
+    },
+    matlabSettingTitle: {
+        id: 'authoring.problemeditor.settings.matlab.title',
+        defaultMessage: 'MATLAB API Key',
+        description: 'Matlab settings card title',
+    },
+    matlabSettingText1: {
+        id: 'authoring.problemeditor.settings.matlab.text.one',
+        defaultMessage: 'Enter the API key provided by MathWorks for accessing the MATLAB Hosted Service. This key is granted for exclusive use by this course for the specified duration.',
+        description: 'Matlab settings card text 1',
+    },
+    matlabSettingText2: {
+        id: 'authoring.problemeditor.settings.matlab.text.two',
+        defaultMessage: 'Please do not share the API key with other courses and notify MathWorks immediately if you believe the key is exposed or compromised. To obtain a key for your course, or to report an issue please contact',
+        description: 'Matlab settings card text 2',
+    },
+    matlabInputLabel: {
+        id: 'authoring.problemeditor.settings.matlab.inputLabel',
+        defaultMessage: 'API Key',
+        description: 'Matlab text input label',
+    },
+    matlabNoKeySummary: {
+        id: 'authoring.problemeditor.settings.matlab.noKeySummary',
+        defaultMessage: 'None',
+        description: 'Matlab no key summary',
+    },
+    resetSettingsTitle: {
+        id: 'authoring.problemeditor.settings.reset.title',
+        defaultMessage: 'Show reset option',
+        description: 'Reset settings card title',
+    },
+    resetSettingsTrue: {
+        id: 'authoring.problemeditor.settings.reset.true',
+        defaultMessage: 'True',
+        description: 'True option for reset',
+    },
+    resetSettingsFalse: {
+        id: 'authoring.problemeditor.settings.reset.false',
+        defaultMessage: 'False',
+        description: 'False option for reset',
+    },
+    resetSettingText: {
+        id: 'authoring.problemeditor.settings.reset.text',
+        defaultMessage: "Determines whether a 'Reset' button is shown so the user may reset their answer, generally for use in practice or formative assessments.",
+        description: 'Reset settings card text',
+    },
+    scoringSettingsTitle: {
+        id: 'authoring.problemeditor.settings.scoring.title',
+        defaultMessage: 'Scoring',
+        description: 'Scoring settings card title',
+    },
+    scoringAttemptsInputLabel: {
+        id: 'authoring.problemeditor.settings.scoring.attempts.inputLabel',
+        defaultMessage: 'Attempts',
+        description: 'Scoring attempts text input label',
+    },
+    scoringWeightInputLabel: {
+        id: 'authoring.problemeditor.settings.scoring.weight.inputLabel',
+        defaultMessage: 'Points',
+        description: 'Scoring weight input label',
+    },
+    scoringSummary: {
+        id: 'authoring.problemeditor.settings.scoring.summary',
+        defaultMessage: "{attempts, plural, =0 {Unlimited} other {#}} attempts - {weight, plural, =0 {Ungraded} other {# points}}",
```

Comment:
Nit: "<=0" instead of "=0" for points and attempts

Reply:
Negative number of attempts are not allowed, that probably can handle it, what do you say? @rayzhou-bit

Reply:
That works too!